lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -706,6 +706,10 @@ namespace Js
 
     Var IntlEngineInterfaceExtensionObject::EntryIntl_IsWellFormedLanguageTag(RecyclableObject* function, CallInfo callInfo, ...)
     {
+#if defined(INTL_ICU)
+        AssertOrFailFastMsg(false, "IsWellFormedLanguageTag is not implemented using ICU");
```

Comment:
>IsWellFormedLanguageTag [](start = 36, length = 23)

Presumably because the implementation was deleted from C++ and moved to JS logic?

Reply:
Correct